sos/plugins/filesys.py

```diff
@@ -25,7 +25,7 @@ class Filesys(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     option_list = [("lsof", 'gathers information on all open files', 'slow',
                     False),
                    ("dumpe2fs", 'dump filesystem information', 'slow', False),
-                   ("frag", 'filesystem fragmentation status', 'slow', False) ]
+                   ("frag", 'filesystem fragmentation status', 'slow', False)]
```

Comment:
This block wasn't formatted properly when you started, but we should make things better, not worse :-)

The accepted style for option lists in `sos` (due to their length) is to have the closing `]` on a line of its own, and (optionally) to place a newline between the opening `[` and the first option opening `(`, i.e.:

```python
    option_list = [
        ("opt1", "description for opt1", "fast", False),
        ("opt2, "description for opt2", "fast", False)
    ]
```

This gives the most space available to the option line, conforms to `pep8`, and makes the list more readable.